### ISSUE TITLE
Maximum call stack size exceeded

### ISSUE THREAD

**Issue by guidouil** [p-j/meteor-crypto-aes#1]
Hi, I'm trying to use this nice package but when I try to encrypt using this on the client side
CryptoJS.AES.encrypt("Message", "Passphrase");
I get the message : Maximum call stack size exceeded
can you help ?
(it might be a noob question)

**Comment by p-j** [comment]
Hi,

Thanks for reporting the issue,
I'll look into it, I'll tell you if I find something.

**Comment by p-j** [comment]
Can you tell me more about how you set it up or point me to a github repository with the source you are using so that I try to reproduce ? 

Starting fresh I wasn't able to, FWIW here is how I tested : 

```
$ meteor create test-aes
$ cd test
$ meteor add jparker:crypto-aes
$ meteor
```

I've added this line to `test-aes.js`

``` javascript
if (Meteor.isClient) {
    //...
    aes = CryptoJS.AES.encrypt("Message", "Passphrase");
}
```

In the browser, I have no errors, in its console, I can inspect the `aes` object.

**Comment by guidouil** [comment]
My bad. I thought it was because of my click event (place where I encrypt the string) but no. I found my mistake thought. It's because of my lazyness...
I was calling the encrypt with the secret like this Meteor.call('get_key', Meteor.userId())
I tried to create a var for it and pass it to the encrypt function and sudently it did work.
Very sorry of using your time for my lazy errors.
Thank you for your reactivity ;)
